### PR TITLE
updated e2e guide for writing

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
Updated instructions for `If test case is failing due to Known Issue` in webapp and mobile e2e guides

image

### CODE REVIEW

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 1]
```diff
@@ -21,14 +21,22 @@ Before writing a script, ensure that it has a corresponding test case in Zephyr.
     ```
     `Zephyr ID` is used for mapping test cases per Release Testing specification. It will be used to measure coverage between manual and automated tests.
 
-3. If a test is failing due to a known issue, append the Jira issue key in the test description, following the format of ` -- KNOWN ISSUE: [Jira_key]`. For example,
-    ```javascript
-    describe('Upload Files', () => {
-        it('MM-T2261 Upload SVG and post -- KNOWN ISSUE: MM-38982', () => {
-            // Test steps and assertion here
+3. If a test is failing due to a known issue, 
+   1. append the Jira issue key in the test description, following the format of ` -- KNOWN ISSUE: [Jira_key]`. For example,
```
Comment (saturninoabril) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:25]:
```suggestion
   1. Append the Jira issue key in the test title, following the format of ` -- KNOWN ISSUE: [Jira_key]`. For example,
```

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 2]
```diff
@@ -21,14 +21,22 @@ Before writing a script, ensure that it has a corresponding test case in Zephyr.
...
+        ```javascript
+        describe('Upload Files', () => {
+            it('MM-T2261 Upload SVG and post -- KNOWN ISSUE: MM-38982', () => {
+                // Test steps and assertion here
+            }
         }
-    }
-    ```
+        ```
+   1. move the test case into a sperate `spec` file, following the format of `<existting spec file name_[1-9].js>`. For example,
```
Comment (saturninoabril) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:33]:
```suggestion
   2. Move the test case into a separate `spec` file following the format of `<existing_spec_file_name_[1-9].js>`. For example,
```

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 3]
```diff
@@ -21,14 +21,22 @@ Before writing a script, ensure that it has a corresponding test case in Zephyr.
...
+     `accessibility_account_settings_spec_1.js` and demote the spec file (i.e remove `// Stage: @prod` from the spec file)
+
+   1. if all the test cases are failing in a spec file, update their names as mentioned above and demote the spec file.
```
Comment (saturninoabril) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:36]:
```suggestion
   3. If all the test cases are failing in a spec file, update each title as mentioned above and demote the spec file.
```

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 4]
```diff
@@ -21,14 +21,22 @@ Before writing a script, ensure that it has a corresponding test case in Zephyr.
...
+
+   1. link the failed test case/s to Jira issue, `Goto Jira Bug > Click Zephyr Scale tab > Click 'add an existing one' link > Select test case/s > Click Add`
```
Comment (saturninoabril) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:38]:
```suggestion
   4. Link the failed test case/s to Jira issue, `Go to Jira Bug > Click Zephyr Scale tab > Click 'add an existing one' link > Select test case/s > Click Add`
```

File: site/content/contribute/webapp/e2e/guide-for-writing.md [hunk 5]
```diff
@@ -34,14 +34,23 @@ Inside the `integration` directory, there are subdirectories that roughly break
...
-    }
-    ```
+        ```
+   1. move the test case into a sperate `spec` file, following the format of `<existting spec file name_[1-9].js>`. For example,
+     `accessibility_account_settings_spec_1.js` and demote the spec file (i.e remove `// Stage: @prod` from the spec file)
+
+   1. if all the test cases are failing in a spec file, update their names as mentioned above and demote the spec file.
+
+   1. link the failed test case/s to Jira issue, `Goto Jira Bug > Click Zephyr Scale tab > Click 'add an existing one' link > Select test case/s > Click Add`
+
```
Comment (saturninoabril) [inline site/content/contribute/webapp/e2e/guide-for-writing.md:52]: Same suggestions as above.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @furqanmlk! Overall looks good to me. Please see suggestions.

**Review by Phrynobatrachus — APPROVED** [review]

**Review by cwarnermm — COMMENTED** [review]
Great update, @furqanmlk! I've provided feedback inline. Please let me know if you have any questions or concerns.

**Review by saturninoabril — APPROVED** [review]
Thanks @furqanmlk! Approving now given all comments are resolved.

**Comment by furqanmlk** [comment]
Thanks @cwarnermm for detailed review

**Review by cwarnermm — APPROVED** [review]
Thanks!

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 6]
```diff
@@ -21,14 +21,22 @@ Before writing a script, ensure that it has a corresponding test case in Zephyr.
...
         }
-    }
-    ```
+        ```
+   2. Move the test case into a separate `spec` file following the format of `<existing_spec_file_name_[1-9].js>`. For example:
+     `accessibility_account_settings_spec_1.js` and demote the spec file (i.e. remove `// Stage: @prod` from the spec file)
+
+   3. If all the test cases are failing in a spec file, update each title as mentioned above and demote the spec file.
+
+   4. Link the failed test case/s to the Jira issue. In the Jira bug, select the **Zephyr Scale** tab. Select the **add an existing one** link, then select test case/s, and select **Add**.
```
Comment (cwarnermm) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:38]:
```suggestion
   4. Link the failed test cases to the Jira issue. In the Jira bug, select the **Zephyr Scale** tab. Select the **add an existing one** link, then select test cases, and select **Add**.
```

**Pull request merged**